entrytool/patient_lists.py

```diff
@@ -5,6 +5,7 @@
 from opal.models import Episode
```

Comment:
I'm not sure we're planning to use these for anything?
Maybe this whole file could have gone in the code cleanup?

Reply:
We are going to keep these in for the time being for dev purposes but will remove before go live.